README.md

```diff
@@ -25,6 +27,12 @@ $ sbt "etl/run <capi key>"
 
 This should run for a few minutes. By the end, you will have a few thousand recipes in your DB.
 
+The next step is then to call the `/admin/prepare-recipes` end point which migrates recipes in `New` status into `Ready`. Note that you need have the play app running to do this. (See "Running play app")
```

Comment:
This is called by me, as part of the manual step by step migration of recipes from `New` to `Ready`. The operation is meant to be idempotent.